web/src/index.tsx

```diff
@@ -45,4 +45,4 @@ const RoutedApp = () => (
     </Router>
 );
 
-ReactDOM.render(<RoutedApp />, document.getElementById('root'));
+createRoot(document.getElementById('root')!).render(<RoutedApp />);
```

Comment:
Am I right TS started to warn this place without this sign? (Because in old interface we didn't have this).

Reply:
Yeap, it's from react documentation